src/test/java/com/github/mmolimar/kafka/connect/fs/task/FsSourceTaskTest.java

```diff
@@ -46,7 +46,7 @@
     );
     private static final int NUM_RECORDS = 10;
     private static final long NUM_BYTES_PER_FILE = 390;
-    private static final String FILE_ALREADY_PROCESSED = "already_processed.txt";
+    private static final String FILE_ALREADY_PROCESSED = "0101010101010101.txt";
```

Comment:
The file regex was inadvertently filtering out this file :whoops. This just renames the file to not be filtered out by the regex. The test still passes.